src/containers/home/index.tsx

```diff
@@ -155,26 +126,43 @@ function Home() {
         {partners && partners.length ? (
           <>
             <div className="partners-carousel-container">
-              <i className="partners-carousel-arrow-left fas fa-chevron-left" onClick={carouselPrev} />
-              <i className="partners-carousel-arrow-right fas fa-chevron-right" onClick={carouselNext} />
-
-              <OwlCarousel
-                responsive={{
-                  0: { items: 2 },
-                  400: { items: 3 },
-                  550: { items: 4 },
-                  800: { items: 5 },
-                  1000: { items: 6 },
-                }}
-                loop
-                smartSpeed={500}
-                onLoad={setCarouselInterval}
-                onDrag={clearCarouselInterval}
-                onDragged={setCarouselInterval}
+              <i className="partners-carousel-arrow-left fas fa-chevron-left" ref={previous} />
+              <i className="partners-carousel-arrow-right fas fa-chevron-right" ref={next} />
+
+              <Swiper
                 className="partners-carousel"
-                ref={carouselRef}>
-                {partners}
-              </OwlCarousel>
+                modules={[Autoplay, Navigation]}
+                loop
+                speed={500}
+                autoplay={{ delay: 2000 }}
```

Comment:
```suggestion
                autoplay={{ delay: 2000, disableOnInteraction: false }}
```
Niveau UX, c'est mieux je trouve